### ISSUE TITLE
Document how to integrate with Sphinx

### ISSUE THREAD

**Issue by johnthagen** [rst2pdf/rst2pdf#802]
**Description of problem**

Currently neither the GitHub README nor the official site (https://rst2pdf.org/) document how to integrate this package with Sphinx.

The best info I could find is: https://stackoverflow.com/a/47720278/1398841

But it seems like this should be documented in the package itself. For example, it's not immediately clear what each of the items in the `pdf_documents` refer to.

**Comment by johnthagen** [comment]
Also, it's not clear (at least to me) how to control the `version` that is put on the title page?

![Screen Shot 2019-09-19 at 9 19 04 AM](https://user-images.githubusercontent.com/10340167/65247449-67b3f880-dabe-11e9-82ee-c175ec9467d0.png)

**Comment by johnthagen** [comment]
It should also document how to use a different index file, which I think I found through search through the code:

https://github.com/rst2pdf/rst2pdf/blob/0e4a9aa1ee10325fb9fc84238ebe8964ba83ed12/rst2pdf/tests/input/sphinx-issue364/conf.py#L204

**Comment by williehallock802** [comment]
> Also, it's not clear (at least to me) how to control the `version` that is put on the cover page?
> 
> ![Screen Shot 2019-09-19 at 9 19 04 AM](https://user-images.githubusercontent.com/10340167/65247449-67b3f880-dabe-11e9-82ee-c175ec9467d0.png)

I would like to know how to control this "version" text as well.

**Comment by akrabat** [comment]
I agree that more documentation in the manual would be great! I'm not a Sphinx user though, so it's not easy for me to do this.

Looking through the tests, I think that to control the version text on the cover page, you add a `version = '1.2.3'` directive to your  your `conf.py`. You can see this in use in the `sphinx-issue183` test's [conf.py](https://github.com/akrabat/rst2pdf/blob/master/rst2pdf/tests/input/sphinx-issue183/conf.py#L47-L48) & [generated PDF](https://github.com/akrabat/rst2pdf/blob/master/rst2pdf/tests/reference/sphinx-issue183.pdf).

The actual text "version x.y" that's on the cover page, appears to be the `subtitle` part of the [default cover page template](https://github.com/akrabat/rst2pdf/blob/master/rst2pdf/templates/sphinxcover.tmpl#L15-L15). You can select your own template using the [`pdf_cover_template` config directive](https://github.com/akrabat/rst2pdf/blob/master/rst2pdf/pdfbuilder.py#L893).

**Comment by yoonghm** [comment]
In sphinx, the following variables in `conf.py` may be useful:

`project` - Title of the document
`author` - Author
`release` - Version of the document

**Comment by lornajane** [comment]
See also #535 and #503 . The maintainers of rst2pdf aren't Sphinx users so any contributions, information or advice from people who do use Sphinx is very welcome.

**Comment by akrabat** [comment]
I didn't point it out earlier, but the manual does have some words on Sphinx integration here https://rst2pdf.org/static/manual.html#sphinx. Clearly we're not surfacing this well though and it is very minimal.
